www/content/workstation/knife_setup.md

```diff
@@ -14,81 +14,49 @@ aliases = ["/knife_setup.html", "/knife_setup/"]
 
-The knife command line tool must be configured to communicate with the
-Chef Infra Server as well as any other infrastructure within your
-organization. This is done initially during the workstation setup, but
-subsequent modifications can be made using the config.rb configuration
-file.
+knife is a command-line tool that provides an interface between a local chef-repo and the Chef Infra Server. The knife command line tool must be configured to communicate with the Chef Infra Server as well as any other infrastructure within your organization.
 
-## config.rb Configuration File
+The first time you set up Chef Infra, you need to manually create the directory for important Chef Infra files, such as `config.rb`.
 
-Knife is configured using a config.rb configuration, which contains
-configuration for both the knife command line tool as well as any
-installed knife plugins. See [config.rb](/workstation/config_rb/) for a complete
-list of configuration options in the config.rb file.
+We recommend setting up knife to use profiles. Knife profiles let you use knife with more than one Chef Infra Server and with more than one organization on a Chef Infra Server.
 
-### Load Path Priority
+To use knife profiles, the first time you set up your workstation enter:
 
-The config.rb file is loaded every time the knife command is invoked
-using the following load order:
-
--   From a specified location given the `--config` flag
--   From a specified location given the `$KNIFE_HOME` environment
-    variable, if set
--   From a `config.rb` file within the current working directory, e.g.,
-    `./config.rb`
--   From a `config.rb` file within a `.chef` directory in the current
-    working directory, e.g., `./.chef/config.rb`
--   From a `config.rb` file within a `.chef` directory located one
-    directory above the current working directory, e.g.,
-    `../.chef/config.rb`
--   From `~/.chef/config.rb` (macOS and Linux platforms) or
-    `c:\Users\<username>\.chef` (Microsoft Windows platform)
+```bash
+mkdir ~/.chef
+touch ~/.chef/credentials
+```
 
-{{< note >}}
+```powershell
+New-Item -Path "c:\" -Name ".chef" -ItemType "directory"
+New-Item -ItemType "file" -Path "c:\.chef\credentials"
+```
 
-When running Microsoft Windows, the config.rb file is located at
-`%HOMEDRIVE%:%HOMEPATH%\.chef` (e.g. `c:\Users\<username>\.chef`). If
-this path needs to be scripted, use `%USERPROFILE%\chef-repo\.chef`.
+Previous Chef Infra setups recommended setting up knife with a `config.rb` file. Configuring knife with `config.rb` is still valid if you work on one Chef Infra Server with one Chef Infra Server organization.
 
-{{< /note >}}
+```bash
+mkdir ~/.chef
+touch ~/.chef/config.rb
+```
 
-### config.rb Configuration Within a Chef Repository
+```powershell
+New-Item -Path "c:\" -Name ".chef" -ItemType "directory"
+New-Item -ItemType "file" -Path "c:\.chef\config.rb"
```

Comment:
```suggestion
knife configure
```

Reply:
See previous comment; `knife configure` is probably not what you want.